hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/model/TotalTargetCountStatus.java

```diff
@@ -153,6 +158,10 @@ private void convertStatus(final TotalTargetCountActionStatus item) {
         case WARNING:
         case DOWNLOAD:
         case DOWNLOADED:
+            if(isDownloadOnly){
```

Comment:
TODO: refactor to method, also:
        case CANCELING:
            if(isDownloadOnly){
                statusTotalCountMap.put(Status.FINISHED, item.getCount());
            } else {
                final Long runningItemsCount = statusTotalCountMap.get(Status.RUNNING) + item.getCount();
                statusTotalCountMap.put(Status.RUNNING, runningItemsCount);
            }
            break;